tests/utils/docker-utils.js

```diff
@@ -0,0 +1,29 @@
+import { execSync } from 'child_process';
+
+const composeFiles = [
+  'docker-compose.yml',
+  'docker-compose.couchdb.yml',
+  'docker-compose.postgres.yml'
+].join(' -f ');
+
+const execDockerCommand = (command) => {
+  try {
+    return execSync(command).toString().trim();
+  } catch (error) {
+    console.error(`Failed to execute command "${command}":`, error);
+    return '';
+  }
+};
+
+export const stopService = (serviceName) => {
+  execDockerCommand(`docker compose --env-file ./tests/.e2e-env -f ${composeFiles} stop ${serviceName}`);
```

Comment:
It's a little strange that we join by `-f` and then we also include a `-f` in the command. 
Can we just prefix the `composeFiles` variable directly, or change the join somehow, so we don't have to use `-f` in the command?